src/components/AccountForm/accountForm.js

```diff
@@ -48,19 +81,31 @@ const AccountForm = ({ accountId = null }) => {
                         //onBlur: () => { },
                         value: formAccount.name
                     },
+                    {
+                        interface: 'input',
+                        type: 'text',
+                        label: 'Descripción de la cuenta',
+                        name: 'description',
+                        onChange: (e) => { updateFormAccount({
+                            ...formAccount,
+                            description: e.target.value
+                        })},
+                        //onBlur: () => { },
+                        value: formAccount.description
+                    },
                     {
                         interface: 'input',
                         type: 'number',
                         label: 'Monto inicial',
-                        name: 'amount',
+                        name: 'balance',
                         onChange: (e) => { updateFormAccount({
                             ...formAccount,
-                            amount: e.target.value
+                            balance: e.target.value
                         })},
                         onBlur: () => { },
-                        value: formAccount.amount
+                        value: formAccount.balance
                     },
-                    {
+                    /*{
```

Comment:
Maybe you coudl remove the commented code that are not providing any extra information